package.json

```diff
@@ -128,7 +128,7 @@
   },
   "main": "index",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4.0.0"
```

Comment:
Travis seems happy with [Node.js v6.1](https://travis-ci.org/adaptlearning/adapt_authoring/jobs/130845301#L280). So we can also make `"node": ">=4.0.0 <6.x"`.

What do you think?